### PR TITLE
INTMDB-311: Feature Add: Prometheus and Microsoft Team to the Third Party Integration Settings

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Feature Add: Prometheus and Microsoft Team to the Third Party Integration Settings

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: mongodbatlas/third_party_integration.go [hunk 1]
```diff
@@ -29,22 +29,29 @@ var _ IntegrationsService = &IntegrationsServiceOp{}
...
+	OrgName                  string `json:"orgName,omitempty"`
+	URL                      string `json:"url,omitempty"`
+	Secret                   string `json:"secret,omitempty"`
+	Name                     string `json:"name,omitempty"`
+	MicrosoftTeamsWebhookURL string `json:"microsoftTeamsWebhookUrl,omitempty"`
+	UserName                 string `json:"username,omitempty"`
+	Password                 string `json:"password,omitempty"`
+	ServiceDiscovery         string `json:"serviceDiscovery,omitempty"`
+	Scheme                   string `json:"scheme,omitempty"`
+	Enabled                  string `json:"enabled,omitempty"`
```
Comment (andreaangiolillo) [inline mongodbatlas/third_party_integration.go:54]: `enabled` is a boolean

File: mongodbatlas/third_party_integration.go [hunk 2]
```diff
@@ -29,22 +29,29 @@ var _ IntegrationsService = &IntegrationsServiceOp{}
...
-	ServiceKey  string `json:"serviceKey,omitempty"`
-	APIToken    string `json:"apiToken,omitempty"`
-	TeamName    string `json:"teamName,omitempty"`
-	ChannelName string `json:"channelName,omitempty"`
-	RoutingKey  string `json:"routingKey,omitempty"`
-	FlowName    string `json:"flowName,omitempty"`
-	OrgName     string `json:"orgName,omitempty"`
-	URL         string `json:"url,omitempty"`
-	Secret      string `json:"secret,omitempty"`
+	Type                     string `json:"type,omitempty"`
```
Comment (andreaangiolillo) [inline mongodbatlas/third_party_integration.go:32]: Can you update the unit test to use these new properties?

**Review by andreaangiolillo — COMMENTED** [review]
Thanks for adding these properties! I have just a few comments.

File: mongodbatlas/third_party_integration.go (hunk 2 above)
Comment (martinstibbe) [reply mongodbatlas/third_party_integration.go:32]: Expanded tests to cover Teams and Prometheus feature added

**Review by andreaangiolillo — APPROVED** [review]
LGTM. Thanks for addressing all the comments! 💯

**Pull request merged**